sdk/communication/Azure.Communication.Chat/tests/samples/Sample3_ParticipantOperations.cs

```diff
@@ -0,0 +1,75 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading.Tasks;
+using Azure.Communication.Administration;
+using Azure.Communication.Administration.Models;
+using Azure.Communication.Identity;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.Communication.Chat.Tests.samples
+{
+    public partial class Sample3_ParticipantOperations : SamplesBase<ChatTestEnvironment>
+    {
+        // This sample demonstrates the operations that can be performed on a thread for participants: add, get and remove participants
+        [Test]
+        public async Task GetAddRemoveMembersAsync()
+        {
+            CommunicationIdentityClient communicationIdentityClient = new CommunicationIdentityClient(TestEnvironment.ConnectionString);
+            Response<CommunicationUser> user1 = await communicationIdentityClient.CreateUserAsync();
+            Response<CommunicationUser> user2 = await communicationIdentityClient.CreateUserAsync();
+            Response<CommunicationUser> user3 = await communicationIdentityClient.CreateUserAsync();
+
+            CommunicationUserToken communicationUserToken1 = await communicationIdentityClient.IssueTokenAsync(user1.Value, new[] { CommunicationTokenScope.Chat });
+            CommunicationUserToken communicationUserToken2 = await communicationIdentityClient.IssueTokenAsync(user2.Value, new[] { CommunicationTokenScope.Chat });
+            CommunicationUserToken communicationUserToken3 = await communicationIdentityClient.IssueTokenAsync(user3.Value, new[] { CommunicationTokenScope.Chat });
+            string userToken = communicationUserToken1.Token;
+            string endpoint = TestEnvironment.ChatApiUrl();
+            string theadCreatorUserId = communicationUserToken1.User.Id;
+
+            ChatClient chatClient = new ChatClient(
+                new Uri(endpoint),
+                new CommunicationUserCredential(userToken));
+
+            var chatParticipant = new ChatParticipant(new CommunicationUser(theadCreatorUserId))
+            {
+                DisplayName = "UserDisplayName",
+                ShareHistoryTime = DateTime.MinValue
+            };
+            ChatThreadClient chatThreadClient = await chatClient.CreateChatThreadAsync(topic: "Hello world!", participants: new[] { chatParticipant });
+            string threadId = chatThreadClient.Id;
+
+            #region Snippet:Azure_Communication_Chat_Tests_Samples_GetParticipants
+            AsyncPageable<ChatParticipant> allParticipants = chatThreadClient.GetParticipantsAsync();
+            await foreach (ChatParticipant participant in allParticipants)
+            {
+                Console.WriteLine($"{participant.User.Id}:{participant.DisplayName}:{participant.ShareHistoryTime}");
+            }
+            #endregion Snippet:Azure_Communication_Chat_Tests_GetMembers
+
+            var participantId1 = theadCreatorUserId;
```

Comment:
Any reason why we take the id out and then recreate the `CommunicationUser`s a little after below?

Reply:
We did that from the beginning just for cosmetics in the readme to show 'participant' in the snippet instead of thread creator or user. I think we are saying based on the previous comment, we could do something like ...  


            #region Snippet:Azure_Communication_Chat_Tests_Samples_AddParticipants
            var participants = new[]
            {
                new ChatParticipant(participant1) { DisplayName ="Josh"},
                new ChatParticipant(participant2) { DisplayName ="Andrew"},
                new ChatParticipant(participant3) { DisplayName ="Peter"}
            }

Reply:
Yes, that would work. My main point is that we do not need to specifically use the string ids. We can always use the `CommunicationUser` objects